dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/UrlMergeProcessor.java

```diff
@@ -14,16 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.dubbo.rpc.cluster.support;
+package org.apache.dubbo.rpc.cluster;
 
 import org.apache.dubbo.common.URL;
 import org.apache.dubbo.common.extension.SPI;
 
 import java.util.Map;
 
-@SPI
-public interface ProviderURLMergeProcessor {
-    URL mergeProviderUrl(URL providerUrl, Map<String, String> localParametersMap);
+@SPI("default")
+public interface UrlMergeProcessor {
```

Comment:
Please avoid changing the SPI definition.

Reply:
done